### DISCUSSION TITLE
clickhouse - No normalization of the raw data

### DISCUSSION THREAD

**Discussion by ashish135906** [airbytehq/airbyte#34396] [Connector Questions]
I am transferring data from MS sql database table to clickhouse table. Data is getting transferred successfully but only in raw json format. I have enabled the default normalization but it is not happening.
[default_workspace_logs_68_txt.txt](https://github.com/airbytehq/airbyte/files/13996947/default_workspace_logs_68_txt.txt)

**Comment by ashish135906** [comment]
2024-01-20 17:29:42 normalization > Unable to connect ClickHouse Connect C to Numpy API [No module named 'numpy'], falling back to pure Python

I see this error.

**Comment by davinchia** [comment]
Hi, this is due to the older version. Please update to the latest Clickhouse version to resolve this. Thanks!
